docs/en/observability/logs-stream.asciidoc

```diff
@@ -389,4 +394,180 @@ Check the following common issues for possible solutions:
+      }
+    }
+  ]
+}
+----
+
+The results should show the `@timestamp` and the `log.level` fields extracted from the `message` field:
+
+[source,JSON]
+----
+{
+  "docs": [
+    {
+      "doc": {
+        "_index": "_index",
+        "_id": "_id",
+        "_version": "-3",
+        "_source": {
+          "message": "192.168.1.101 Disk usage exceeds 90%.",
+          "log": {
+            "level": "WARN"
+          },
+          "@timestamp": "2023-8-08T13:45:12.123Z",
+        },
+        ...
+      }
+    }
+  ]
+}
+----
+
+[discrete]
+[[logs-stream-log-level-query]]
+=== Query logs based on `log.level`
+
+With the `log.level` field extracted, you can query high-severity logs (`WARN` and `ERROR`) that might require your immediate attention, and not be overwhelmed by low-severity logs (`INFO` and `DEBUG`) that aren't immediately important. 
+
+Let's say you have the following logs with varying severities:
+
+[source,log]
+----
+2023-08-08T13:45:12.123Z WARN 192.168.1.101 Disk usage exceeds 90%.
+2023-08-08T13:45:14.003Z ERROR 192.168.1.103 Database connection failed.
+2023-08-08T13:45:15.004Z DEBUG 192.168.1.104 Debugging connection issue.
+2023-08-08T13:45:16.005Z INFO 192.168.1.102 User changed profile picture.
+----
+
+You can add them to your data stream using this command:
+
+[source,console]
+----
+POST logs-example-default/_bulk
+{ "create": {} }
+{ "message": "2023-08-08T13:45:12.123Z WARN 192.168.1.101 Disk usage exceeds 90%." }
+{ "create": {} }
+{ "message": "2023-08-08T13:45:14.003Z ERROR 192.168.1.103 Database connection failed." }
+{ "create": {} }
+{ "message": "2023-08-08T13:45:15.004Z DEBUG 192.168.1.104 Debugging connection issue." }
+{ "create": {} }
+{ "message": "2023-08-08T13:45:16.005Z INFO 192.168.1.102 User changed profile picture." }
+----
+
+The following command queries your documents based on the log level keyword and only shows those with `WARN` or `ERROR` log levels: 
+
+[source,console]
+----
+GET logs-example-default/_search
+{
+  "query": {
+    "terms": {
+      "log.level.keyword": ["WARN", "ERROR"]
+    }
+  }
+}
+----
+
+You should see the following results with only high-severity logs:
+
+[source,JSON]
+----
+{
+...
+  },
+  "hits": {
+  ...
+    "hits": [
+      {
+        "_index": ".ds-logs-example-default-2023.08.14-000001",
+        "_id": "3TcZ-4kB3FafvEVY4yKx",
+        "_score": 1,
+        "_source": {
+          "message": "192.168.1.101 Disk usage exceeds 90%.",
+          "log": {
+            "level": "WARN"
+          },
+          "@timestamp": "2023-08-08T13:45:12.123Z"
+        }
+      },
+      {
+        "_index": ".ds-logs-example-default-2023.08.14-000001",
+        "_id": "3jcZ-4kB3FafvEVY4yKx",
+        "_score": 1,
+        "_source": {
+          "message": "192.168.1.103 Database connection failed.",
+          "log": {
+            "level": "ERROR"
+          },
+          "@timestamp": "2023-08-08T13:45:14.003Z"
+        }
+      }
+    ]
+  }
+}
+----
```

Comment:
Follow up request (out of scope of this PR): Show the user how to do a terms agg on the log.level field to see how many docs there are for a certain time period.